packages/flutter/lib/src/widgets/framework.dart

```diff
@@ -1116,6 +1156,10 @@ abstract class SingleChildRenderObjectWidget extends RenderObjectWidget {
 /// storage for that child list, it doesn't actually provide the updating
 /// logic.)
 abstract class MultiChildRenderObjectWidget extends RenderObjectWidget {
+  /// Initializes this objects fields for its subclasses.
```

Comment:
object's

This is an interesting phrasing. Have we used it elsewhere to document abstract non-const constructors?